components/base/Header.vue

```diff
@@ -38,17 +50,32 @@
   height: 80px;
   line-height: 80px;
   padding: 0px 5%;
-  border-bottom-style: solid; 
+  border-bottom-style: solid;
   border-bottom-width: 0.5px;
   border-bottom-color: $gray;
   ul {
     display: flex;
     justify-content: flex-end;
   }
-  .pc-header-item {
-    margin-left: 90px;
-    list-style-type: none;
+  .pc-header{
+    &-item {
+      margin-left: 90px;
+      list-style-type: none;
+    }
+    &-nested{
+      position: absolute;
+      top:85px;
```

Comment:
展開されたときにヘッダーとくっつかないように、敢えて80pxではなく5px多めに取っておいていただいていると思うんですが、逆にカーソルが間の5pxに差し掛かったときに閉じられてしまって一向にnested-itemをクリックできない(めっちゃ早く動かせば実はできる)ので、なんとかしたいです。
が、ヘッダーの背景・文字色にも寄ると思うので、今がんばって対応するよりヘッダー背景色を変更するときに解決するのはいかがでしょうか🙇‍
見栄えが悪くてもちゃんとクリックできる方がうれしいかなと思うので、一旦くっつけてしまってもよいかなと思いました🙇‍
```suggestion
      top: 80px;
```

Reply:
#128

Reply:
a286709
ちょっと工夫してみました。
こんな感じでいかがでしょう